CHANGELOG.rst

```diff
@@ -19,7 +31,7 @@ v5.0.0
 Release Summary
 ---------------
 
-Starting from this release, the minimum `ansible-core` version this collection requires is `2.14.0`. The last known version compatible with ansible-core<2.14 is `v4.0.3`.
+Starting from this release, the minimum `ansible-core` version this collection requires is `2.14.0`. The last known version compatible with
```

Comment:
this seems to be unintended

Reply:
yea that was due to antsibull, reverting that.